arch/arc/Kconfig

```diff
@@ -89,6 +89,14 @@ config CPU_HS3X
 	help
 	  If y, the SoC uses an ARC HS3x or HS4x CPU
 
+config CPU_HS4X
```

Comment:
We should add default mcpu value for hs4x as we are adding new cpu type.

Otherwise mcpu would be empty (if we don't use mcpu overwrite feature).

IOW: It would work for this board, but we should provide reasonable default (mcpu=hs4x)

Reply:
Done. Ref. cmake/gcc-m-cpu.cmake